### PR TITLE
BATCH statements over QUERY protocol tests

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: shotover-proxy/src/frame/cassandra.rs [hunk 1]
```diff
@@ -60,8 +63,8 @@ pub(crate) fn metadata(bytes: &[u8]) -> Result<CassandraMetadata> {
     })
 }
 
-/// Count queries only from an unparsed Cassandra frame
-pub(crate) fn get_message_count(bytes: &[u8]) -> Result<NonZeroU32> {
+/// Count "cells" only from an unparsed Cassandra frame
+pub(crate) fn cell_count(bytes: &[u8]) -> Result<NonZeroU32> {
```
Comment (rukai) [inline shotover-proxy/src/frame/cassandra.rs:67]: Looks like you messed up a merge or something?

Comment (conorbros) [reply shotover-proxy/src/frame/cassandra.rs:67]: No, I just branched off #574 so I can add onto the existing batch statement tests.

Comment (rukai) [reply shotover-proxy/src/frame/cassandra.rs:67]: oh huh, I forgot that was still living in another PR.
ben plz

**Review by rukai — APPROVED** [review]
lgtm

**Review by benbromhead — APPROVED** [review]

**Pull request merged**